### PR TITLE
RSC-2 RSC-4 Fixes for misc ReviewWidget alignments

### PR DESCRIPTION
**Pull request by arohachenko**

### CODE REVIEW

File: src/css/skins/printdeal/themes/fr-responsive/objects/_breadcrumb.scss [hunk 1]
```diff
@@ -3,3 +3,9 @@
 .Breadcrumb {
     margin-bottom: $base-spacing-unit;
 }
+
+.Flag__Body {
```
Comment (bartveneman) [inline src/css/skins/printdeal/themes/fr-responsive/objects/_breadcrumb.scss]: You could use '.Trim' to cancel the margin-bottom on the '.Breadcrumb`

File: src/css/skins/printdeal/themes/fr-responsive/objects/_review-widget.scss [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+            padding-left: 0.4em;
+        }
+    }
+
+    a {
+        text-decoration: none;
+    }
+
+    img {
+        max-width: none;
```
Comment (bartveneman) [inline src/css/skins/printdeal/themes/fr-responsive/objects/_review-widget.scss]: Have you tested the responsive behaviour on this?

File: src/css/skins/printdeal/themes/fr-responsive/objects/_breadcrumb.scss (hunk 1 above)
Comment (arohachenko) [reply src/css/skins/printdeal/themes/fr-responsive/objects/_breadcrumb.scss]: Thanks, I'll use that instead.

File: src/css/skins/printdeal/themes/fr-responsive/objects/_review-widget.scss (hunk 2 above)
Comment (arohachenko) [reply src/css/skins/printdeal/themes/fr-responsive/objects/_review-widget.scss]: Images in there are just small logos, and the whole block is `show-for-medium-up` anyway, so it's fine.

**Pull request merged**